diff --git a/miLazyCrackerFreshInstall.sh b/miLazyCrackerFreshInstall.sh
index 0c86797..7516b89 100755
--- a/miLazyCrackerFreshInstall.sh
+++ b/miLazyCrackerFreshInstall.sh
@@ -7,32 +7,6 @@ fi
 
 set -x
 
-# run this from inside miLazyCracker git repo
-if [ -f "/etc/debian_version" ]; then
-    pkgs=""
-    for pkg in git libnfc-bin autoconf libnfc-dev; do
-        if ! dpkg -l $pkg >/dev/null 2>&1; then
-            pkgs="$pkgs $pkg"
-        fi
-    done
-    if [ "$pkgs" != "" ]; then
-        sudo apt-get install $pkgs
-    fi
-fi
-
-# install MFOC
-[ -d mfoc ] || git clone https://github.com/nfc-tools/mfoc.git
-(
-    cd mfoc || exit 1
-    git reset --hard
-    git clean -dfx
-    # tested against commit 9d9f01fb
-    autoreconf -vfi
-    ./configure
-    make
-    sudo make install
-)
-
 # install Hardnested Attack Tool
 [ -d crypto1_bs ] || git clone https://github.com/aczid/crypto1_bs
 (
@@ -45,9 +19,9 @@ fi
     mkdir crapto1-v3.3
     tar Jxvf ../crapto1-v3.3.tar.xz -C crapto1-v3.3
     make
-    sudo cp -a libnfc_crypto1_crack /usr/local/bin
+    install -Dm755 libnfc_crypto1_crack "$DESTDIR/usr/local/bin/libnfc_crypto1_crack"
 )
 
 # install our script
-sudo cp -a miLazyCracker.sh /usr/local/bin/miLazyCracker
+install -Dm755 miLazyCracker.sh "$DESTDIR/usr/local/bin/miLazyCracker"
 echo "Done."
